apps/vscode/command_client/command_client.py

```diff
@@ -13,10 +13,10 @@
 # to remove it
 STALE_TIMEOUT_MS = 60_000
 
-# The amount of time to wait for VSCode to perform a command, in seconds
+# The amount of time to wait for client application to perform a command, in seconds
 VSCODE_COMMAND_TIMEOUT_SECONDS = 3.0
```

Comment:
You could change this name to no longer include VSCODE

Reply:
Yes missed this sorry.